app/main.ipynb

```diff
@@ -31,15 +35,31 @@
+     },
+     "execution_count": 5,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
    "source": [
-    "# write your code for task 0 here"
+    "data = pd.read_csv(\"C:/projects/py-adult-data-analysis/data/adult.csv\")\n",
+    "clean_data = data[~data.isin([\"?\"]).any(axis=1)]\n",
+    "clean_data"
```

Comment:
did you check data correctness "Also check for data correctness (salary & salary $K)."?